design/one-pager-release-process.md

```diff
@@ -0,0 +1,321 @@
+less wait time. Contributors will have more frequent opportunities to include their work into a
+release, which will decrease the stress and pressure we currently feel leading up to each large and
+long awaited release.
+
+We propose that a desirable frequency of release will be **monthly**, as that will provide a
+predictable and consistent schedule that gets new features out to the community at a reasonable
+pace.  We currently execute on work items in a series of sprints, each lasting 2 weeks.  To
+accommodate a monthly release schedule, this sprint schedule will need to be updated.
+
+As part of the timeline that we use to release software, there are many other events along the way
+besides the release itself that facilitate a healthy software development lifecycle. A brief
+description of these events and their associated goals and expectations can be found below:
+
+* **Milestone planning**: Before a milestone begins, we need to discuss the high level priorities
+  and themes, and identify the epics that will deliver on these priorities.
+* **Epic discovery**: After epics are identified, we need to do some discovery and investigation
+  work to appropriately scope the epics, identify the risks, and agree upon the epic's definition of
+  done.
+* **Weekly planning**: On a weekly basis, the team can get together for a more focused status check
+  with potential course correcting.  We can identify if any priorities have changed and if any
+  resources or efforts need to be reorganized. At the end of this meeting, everyone should have a
+  clear understanding of what their work will focus on for the week. This is similar to the goal of
+  sprint planning previously.
+* **Stand-up scrums**: This is a daily event where we quickly meet to talk about our progress, if we
+  are blocked on anything, and if we need to have any break-out sessions to help work through
+  issues.
+* **Backlog grooming and triage**: The backlog of issues and work items needs to be addressed and
+  explored on a regular basis, especially in response to user demand.  In this session, we can
+  triage new issues to understand their priority, we can revisit items in the backlog to see if any
+  priorities have changed, and we can work on the epic discovery mentioned above if that also needs
+  attention.
+* **Demo party**: This is a chance to celebrate and show off the fruits of our recent labors to the
+  rest of the team.  Each developer can show a demo of new things they have been working on to
+  collect feedback and disseminate information about our progress and limitations to the rest of the
+  team.
+* **Retrospective**: After a milestone is completed, a retrospective look back over it can be very
+  helpful at identifying where we were successful and what we need to improve on.  This retro should
+  result in a series of action items that will help us improve our processes.
```

Comment:
What is the recommendation for tracking retrospective action items and their progress?

Reply:
I think we should do a better job of opening issues for tracking action items.  I'll add a note for that.